src/indicator/algorithm/calculator/stochasticOscillator.js

```diff
@@ -56,13 +56,8 @@ export default function() {
         return stochastic;
     };
 
-    rebind(stochastic, kWindow, {
-        kWindowSize: 'windowSize'
-    });
-
-    rebind(stochastic, dWindow, {
-        dWindowSize: 'windowSize'
-    });
+    rebindAll(stochastic, kWindow, map({'windowSize': 'kWindowSize'}));
+    rebindAll(stochastic, dWindow, map({'windowSize': 'dWindowSize'}));
```

Comment:
`kWindow` and `dWindow` both have an `accumulator` property, now that you `rebindAll` for both, what happens with this collision? (Similar comment for other components in this namespace)